types/pdfkit/index.d.ts

```diff
@@ -121,6 +134,7 @@ declare namespace PDFKit.Mixins {
         strokeOpacity(opacity: number): this;
         linearGradient(x1: number, y1: number, x2: number, y2: number): PDFLinearGradient;
         radialGradient(x1: number, y1: number, r1: number, x2: number, y2: number, r2: number): PDFRadialGradient;
+        pattern(bbox: [number, number, number, number], xStep: number, yStep: number, stream: string): PDFTilingPattern;
```

Comment:
No requirement but instead of directly use [number, number, number, number] you could declare a type (e.g., BoundingBox) as it would improve the readability.